Sources/PackageModel/Manifest/PackageDependencyDescription.swift

```diff
@@ -12,19 +12,19 @@ import Foundation
 import TSCBasic
 
 /// Represents a package dependency.
-public enum PackageDependency: Equatable {
+public enum PackageDependency: Equatable, Hashable {
```

Comment:
Hashable implies Equatable, but it makes sense to me as a style thing to list both.  I think I've been doing just Hashable, should we switch to consistently listing both in this code base going forward?  I kind of prefer it for clarity now that I see it.

Reply:
its a style thing. we have a mixed bag in the code base today. probably topic for a cleanup PR